### PR TITLE
PositionMgt User cases

### PR DESCRIPTION
**Pull request by staceylry**
plz point out any problem that i can improve, thx!!

### CODE REVIEW

**Review by kaizen-w — CHANGES_REQUESTED** [review]
The basic structure looks ok. One suggestion I can add is that the gateway should check if the employee is in the Database. Also, I believe that you are missing import statements. I'm also a little confused as to how the save method works. I believe that you are trying to save data in the Interactor?  I think that your goal here was to save data in the Response Model to the Database. The request model should be created by the controller rather than the Interactor and it would be better to create a response model in the Interactor, pass it to the gateway to save the data in the response in the database.

**Review by zeyuzhang516 — APPROVED** [review]

**Review by tlan4 — CHANGES_REQUESTED** [review]

File: src/main/java/PositionMgt/PositionMgtInputBoundary.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package PositionMgt;
+
+import Entity.User;
+
+public interface PositionMgtInputBoundary {
+    PositionMgtRequestModel create(PositionMgtRequestModel requestModel);
```
[the remark this answers is not in the document]
Comment (tlan4) [reply src/main/java/PositionMgt/PositionMgtInputBoundary.java:6]: As in the return type should response model

[the remark this answers is not in the document]
Comment (staceylry) [reply src/main/java/PositionMgt/PositionMgtInputBoundary.java]: thx! I just fixed this problem

**Comment by staceylry** [comment]
> The basic structure looks ok. One suggestion I can add is that the gateway should check if the employee is in the Database. Also, I believe that you are missing import statements. I'm also a little confused as to how the save method works. I believe that you are trying to save data in the Interactor? I think that your goal here was to save data in the Response Model to the Database. The request model should be created by the controller rather than the Interactor and it would be better to create a response model in the Interactor, pass it to the gateway to save the data in the response in the database.

thx! I just modified this to make it better

**Review by KunLW — APPROVED** [review]

**Review by kaizen-w — APPROVED** [review]

**Review by zeyuzhang516 — APPROVED** [review]

**Review by lungerjo — APPROVED** [review]

**Pull request closed without merging**